e2e-tests/src/main/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java

```diff
@@ -691,21 +808,9 @@ public void getTableInfoWithBadlyQualifiedTableGives404AndMessageAndTraceId() th
         assertThat(info.getErrors().get(0).getStatus(), equalTo(404));
     }
 
-    @Test
-    public void getTableInfo_should_returnTableAndSchema() throws Exception {
-        Table tableInfo = dataConnectApiGetRequest("/table/" + trinoPaginationTestTable + "/info", 200, Table.class);
-        assertThat(tableInfo, not(nullValue()));
-        assertThat(tableInfo.getName(), equalTo(trinoPaginationTestTable));
-        assertThat(tableInfo.getDataModel(), not(nullValue()));
-        assertThat(tableInfo.getDataModel().getId(), not(nullValue()));
-        assertThat(tableInfo.getDataModel().getSchema(), not(nullValue()));
-        assertThat(tableInfo.getDataModel().getProperties(), not(nullValue()));
-        assertThat(tableInfo.getDataModel().getProperties().entrySet(), not(empty()));
-    }
-
     @Test
     public void getTableData_should_returnDataAndDataModel() throws Exception {
-        Table tableData = dataConnectApiGetRequest("/table/" + trinoPaginationTestTable + "/data", 200, Table.class);
+        Table tableData = dataConnectApiGetRequest("/table/" + trinoPaginationTestTableName + "/data", 200, Table.class);
         assertThat(tableData, not(nullValue()));
         tableData = dataConnectApiGetAllPages(tableData);
```

Comment:
Here's one more instance where `tableData =` could be removed.

If you've eliminated all occurrences of this, maybe it's a good idea to change the return type of `dataConnectApiGetAllPages()` to `void`. That would make it clearer that the method modifies the argument rather than returning a new result object and leaving the argument unmodified.